hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VElement<{}>[]
+): VNode<Data>
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  ...children: VElement<{}>[]
+): VNode<Data>
+
```

Comment:
One more thing. 
Can I use `h()` with a number as children ?

plus: do you prefer the `Array<Data>` notation instead of the `Data[]` one ?